config/environments/production.rb

```diff
@@ -87,4 +87,7 @@
       domain: "heroku.com",
       enable_starttls_auto: true
   }
+
+  config.action_mailer.default_url_options = {
+  host: "tipsy-tally-staging.herokuapp.com" }
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.